lessons/r/ggplot2/ggplot2.Rmd

```diff
@@ -7,18 +7,17 @@ output: pdf_document
   
 > ### Learning Objectives
 >
-> - Produce scatter plots, line plots, histograms, bar plots,
-> and boxplots using ggplot.
-> - Plot model fits with `geom_smooth`.
-> - Set universal plot settings.
-> - Understand and apply faceting in ggplot.
-> - Customize plots with `theme`.
+> Create standard plots with `ggplot2` (scatter, line, hist)
+> Learn to modify plot appearance with `theme()`
+* Faceting plots
+> Composing multiple plots with `patchwork`
+> Annotating plots (model fits + text)
+>
 > 
 > ### Setup/Required packages
 >
-> - `install.packages('dplyr')`
-> - `install.packages('ggplot2')`
-> - `install.packages('knitr')`
+> - `install.packages('tidyverse')`
+> - `install.packages('patchwork')`
```

Comment:
Does this work now? Thought it had to be install through GitHub using Devtools?

Reply:
Yep, I checked yesterday and it's officially been on CRAN as of last month!